test/stress/n_workers_reputers_stress_test.go

```diff
@@ -0,0 +1,314 @@
+		//    topics left before the max to reach topicsPerTopicIteration
+		if topicCount+topicsPerTopicIteration > topicsMax {
+			topicsThisEpoch = topicsMax - topicCount
+		}
+		for j := 0; j < topicsThisEpoch; j++ {
+			// 4. Get ahold of the funder for this topic
+			topicFunderAccountName := getTopicFunderAccountName(topicCount)
+			funder := topicFunders[topicFunderAccountName]
+
+			wg.Add(1)
+			// 5. call the inner worker reputer loop that will create
+			// reputers and workers for this topic and push data
+			// to the chain for this topic
+			go workerReputerLoop(
+				&wg,
+				m,
+				funder,
+				reputersPerIteration,
+				maxReputersPerTopic,
+				workersPerIteration,
+				maxWorkersPerTopic,
+				maxWorkerReputerIterations,
+				epochLength,
+				makeReport,
+			)
+			topicCount++
+		}
+
+		// 5. Sleep for enough time to let an epoch to complete before making the next topic
+		elapsedIteration := time.Since(startIteration)
+		sleepingTime := iterationTime - elapsedIteration
+		m.T.Log(topicLog(uint64(topicCount), time.Now(), "Main loop sleeping", sleepingTime))
+		time.Sleep(sleepingTime)
+	}
+	m.T.Log("All routines launched: waiting for running routines to end.")
+	wg.Wait()
+
+	// 6. If applicable, generate a summary report of the test
+	if makeReport {
+		reportSummaryStatistics()
+	}
+}
+
+// Main worker-reputer per-topic loop
+// inside each topic, take the following actions
+// Each call of the workerReputerLoop corresponds to ONE topic
+func workerReputerLoop(
+	wg *sync.WaitGroup,
+	m testCommon.TestConfig,
+	funder AccountAndAddress,
+	reputersPerIteration,
+	maxReputersPerTopic,
+	workersPerIteration,
+	maxWorkersPerTopic,
+	maxIterations,
+	epochLength int,
+	makeReport bool,
+) {
+	defer wg.Done()
+	approximateSecondsBlockTime := getApproximateBlockTimeSeconds(m)
+
+	//  1. Create and fund a new topic
+	//     Additionally register the topic funder as a worker and reputer for that topic
+	topicId := setupTopic(m, funder, int64(epochLength))
+
+	//  2. Generate the worker and reputer bech32 accounts we will need for this new topic
+	numWorkersInThisTopic := min(maxIterations*workersPerIteration, maxWorkersPerTopic)
+	numReputersInThisTopic := min(maxIterations*reputersPerIteration, maxReputersPerTopic)
+	workers := createWorkerAddresses(m, topicId, numWorkersInThisTopic)
+	reputers := createReputerAddresses(m, topicId, numReputersInThisTopic)
+
+	//  3. Fund the accounts
+	err := fundAccounts(
+		m,
+		funder,
+		workers,
+		initialWorkerReputerFundAmount,
+	)
+	if err != nil {
+		m.T.Log(topicLog(topicId, "Error funding worker accounts: ", err))
+	} else {
+		m.T.Log(topicLog(topicId, "Funded", len(workers), "worker accounts."))
+	}
+	err = fundAccounts(
+		m,
+		funder,
+		reputers,
+		initialWorkerReputerFundAmount,
+	)
+	if err != nil {
+		m.T.Log(topicLog(topicId, "Error funding reputer accounts: ", err))
+	} else {
+		m.T.Log(topicLog(topicId, "Funded", len(reputers), "reputer accounts."))
+	}
```

Comment:
Just combine these into a single call with both lists, each call to the chain takes a long time espeically when compounded across an entire topic stress test